geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/scala/org/locationtech/geomesa/fs/storage/common/FileSystemThreadedReader.scala

```diff
@@ -212,8 +224,6 @@ object FileSystemThreadedReader extends StrictLogging {
       iter: => CloseableIterator[SimpleFeature]
     ) extends Runnable {
 
-    phaser.register()
```

Comment:
I think I followed the other Phasers.  Where did this one go?

Reply:
went up to line 203 here: https://github.com/locationtech/geomesa/pull/2713/files/eaf4f4eb2535bc5b12861df6146b4bf398ba4e06#diff-72f9fb4eace5357104fd4f3412a896d6891987d26b26e47b9e4940de76e51126R203

I thought it was clearer to register the phaser where it's created, instead of using the constructor of the reader to register it.